mojoshader.h

```diff
@@ -3349,32 +3350,23 @@ typedef struct MOJOSHADER_mtlShader MOJOSHADER_mtlShader;
  *  (malloc_d) parameter. This pointer is passed as-is to your (m) and (f)
  *  functions.
  *
- * The context created by this function will automatically become the current
- *  context. No further action is needed by the caller.
- *
- * Returns 0 on success or -1 on failure.
+ * Returns a new context on success, NULL on error. If you get a new context,
+ *  you need to make it current before using it with
+ *  MOJOSHADER_mtlMakeContextCurrent().
  */
-DECLSPEC int MOJOSHADER_mtlCreateContext(void *mtlDevice, int framesInFlight,
-                                         MOJOSHADER_malloc m, MOJOSHADER_free f,
-                                         void *malloc_d);
+DECLSPEC MOJOSHADER_mtlContext *MOJOSHADER_mtlCreateContext(void *mtlDevice,
+                                        int framesInFlight, MOJOSHADER_malloc m,
```

Comment:
Are we still doing the framesInFlight stuff, or do we want to throw that idea out for sure?

Other than that this lgtm.

Reply:
Oh whoops, good catch. I'll fix that.